test/scenarios-bootc/presubmits/el94-src@ipv6.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# Redefine network-related settings to use the dedicated IPv6 network bridge
+# shellcheck disable=SC2034  # used elsewhere
+VM_BRIDGE_IP="$(get_vm_bridge_ip "${VM_IPV6_NETWORK}")"
+# shellcheck disable=SC2034  # used elsewhere
+WEB_SERVER_URL="http://[${VM_BRIDGE_IP}]:${WEB_SERVER_PORT}"
+# shellcheck disable=SC2034  # used elsewhere
+MIRROR_REGISTRY_URL="${VM_BRIDGE_IP}:${MIRROR_REGISTRY_PORT}"
```

Comment:
We probably also need to fix the registry URL by adding `[]`, or we're getting the following error.
```
18:26:16,940 INF program: Running... ostree container image deploy --sysroot=/mnt/sysimage --image=2001:db8:ca7:ff::1:5000/rhel94-bootc-source --transport=registry
18:26:17,203 INF program: error: Performing deployment: Creating importer: Failed to invoke skopeo proxy method OpenImage: remote error: invalid reference format
```

Reply:
Not enough, skopeo [does not work with ipv6](https://github.com/containers/image/issues/1004) addresses so I will try to see if /etc/hosts resolution works. Also asked the bootc team about their recommendation so this is just a test to verify my understanding.